packages/itwinui-css/src/bridge.scss

```diff
@@ -245,35 +282,117 @@
     }
   }
 
+  /* ---------------------------------------------------------------------------- */
+  /* Icon */
+
+  .iui-svg-icon {
+    // StrataKit icons are text color by default
+    --_iui-svg-icon-default-fill: currentColor;
```

Comment:
I would leave this one out from this PR. StrataKit `Icon` is currently only catering to the use cases where it is nested inside another component that overrides its fill. We don't usually want `currentColor` to be used; it's just a fallback.